<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPlayer</class>
 <widget class="QWidget" name="MainPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainPlayer</string>
  </property>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>238</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>238</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QListWidget" name="playList">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>40</y>
     <width>256</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>11</y>
     <width>156</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select File to Play</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="audioFileList"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="showError">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Codec not supprted for AAC. Please stop and select other file.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
